.github/workflows/test.yml

```
@@ -16,7 +16,7 @@ jobs:
       max-parallel: 5
       matrix:
         python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
-        django-version: ['4.2', '5.0', '5.1']
+        django-version: ['4.2', '5.0', '5.1', '5.2']
         drf-version: ['3.14', '3.15']
```

```suggestion
        django-version: ['4.2', '5.0', '5.1', '5.2']
        drf-version: ['3.14', '3.15', '3.16']
```
Since Django 5.2 is supported only by DRF 3.16 would you add it and update also the tox.ini file?